forcing_tools/netcdf2vic.py

```diff
@@ -0,0 +1,372 @@
+            self.f.close()
+        except:
+            pass
+
+    def __str__(self):
+        return "Point({0},{1},{2},{3})".format(self.lat, self.lon,
+                                               self.y, self.x)
+
+    def __repr__(self):
+        return "Point(lat={0}, lon={1}, \
+                      y={2}, x={3}, \
+                      filename={4})".format(self.lat,
+                                            self.lon,
+                                            self.y,
+                                            self.x,
+                                            self.filename)
+
+# -------------------------------------------------------------------- #
+
+
+# -------------------------------------------------------------------- #
+def main():
+    config_dict = process_command_line()
+
+    points, ncfiles = init(config_dict)
+
+    points = run(config_dict, ncfiles, points)
+
+    final(config_dict, ncfiles, points)
+# -------------------------------------------------------------------- #
+
+
+# -------------------------------------------------------------------- #
+def init(config_dict):
+
+    # ---------------------------------------------------------------- #
+    # Read domain file
+    domain_dict = config_dict['DOMAIN']
+    dom_data, dom_vatts, dom_gatts = read_domain(config_dict['DOMAIN'])
+    ys, xs = np.nonzero(dom_data[domain_dict['MASK_VAR']])
+    # ---------------------------------------------------------------- #
+
+    # ---------------------------------------------------------------- #
+    # Make sure OUTPATH is available
+    if not os.path.exists(config_dict['OPTIONS']['OUTPATH']):
+        os.makedirs(config_dict['OPTIONS']['OUTPATH'])
+    # ---------------------------------------------------------------- #
+
+    # ---------------------------------------------------------------- #
+    # Setup Points
+    points = []
+    for y, x in zip(ys, xs):
+        points.append(Point(y=y,
+                            x=x,
+                            lat=dom_data[domain_dict['LATITUDE_VAR']][y, x],
+                            lon=dom_data[domain_dict['LONGITUDE_VAR']][y, x],
+                            prefix=config_dict['OPTIONS']['PREFIX'],
+                            griddecimal=config_dict['OPTIONS']['GRIDDECIMAL'],
+                            outpath=config_dict['OPTIONS']['OUTPATH']))
+    points = points[:100]
+    # Add fileformat specific attributes and assign open and write functions
+    if config_dict['OPTIONS']['FILEFORMAT'].lower() == 'ascii':
+        float_format = "%1.{0}f".format(config_dict['OPTIONS']['FLOAT_PREC'])
+        for p in points:
+            p._open_ascii()
+            p.append = p._append_ascii
+            p.float_format = float_format
+    elif config_dict['OPTIONS']['FILEFORMAT'].lower() == 'binary':
+        raise ValueError('Binary output is not yet complete, use ascii')
+        for p in points:
+            p._open_binary()
+            p.append = p._append_binary
```

Comment:
Confused what is going on here - you're raising a `ValueError` but still looping over points?